app/src/main/java/ie/macinnes/tvheadend/account/AuthenticatorActivity.java

```diff
@@ -420,10 +420,10 @@ public void onAuthenticationStateChange(@NonNull Authenticator.State state) {
                 fragment.setArguments(getArguments());
                 add(getFragmentManager(), fragment);
             } else {
-                Log.w(TAG, "Failed to validate credentials");
+                Log.w(TAG, getString(R.string.account_credential_error));
```

Comment:
I'd agree with you. I thought I left out all the log entries, but must have got a bit overzealous